PLMS/Models/TaskCommentModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace PLMS.API.Models
+{
+    public class TaskCommentModel
+    {
```

Comment:
add Id. Since this is a view model we return in task view models, it should contain id for further interaction with comments.